GVRf/Framework/framework/src/main/res/raw/phong_surface_layertex.fsh

```diff
@@ -45,17 +50,18 @@ layout(location = 14) in vec2 emissive_coord1;
 layout(set = 0, binding = 15) uniform sampler2D emissiveTexture1;
 #endif
 
-#ifdef HAS_lightMapTexture1
-in vec2 lightmap_coord1;
+#ifdef HAS_lightmapTexture1
+layout(location = 15) in vec2 lightmap_coord1;
+layout(set = 0, binding = 15) uniform sampler2D lightmapTexture1;
 #endif
 
 
-layout(set = 0, binding = 6) uniform sampler2D ambientTexture;
-layout(set = 0, binding = 5) uniform sampler2D diffuseTexture;
-layout(set = 0, binding = 7) uniform sampler2D specularTexture;
-layout(set = 0, binding = 8) uniform sampler2D opacityTexture;
-layout(set = 0, binding = 9) uniform sampler2D lightmapTexture;
-layout(set = 0, binding = 11) uniform sampler2D emissiveTexture;
+layout(set = 0, binding = 4) uniform sampler2D diffuseTexture;
```

Comment:
Should start from 5.

Reply:
I have changed it to start the texture bindings at 10. This is consistent with the statesort branch in which this is already changed.